### PR TITLE
Add example scripts to help with onboarding

### PR DESCRIPTION
**Pull request by n1tranquilla**

### CODE REVIEW

File: scripts/download-blocks.sh
Comment (jhult) [inline scripts/download-blocks.sh:17]: I think we can/should get rid of `gsutil` and download from our own bucket

See https://github.com/Granola-Team/mina-indexer/issues/211#issuecomment-2010888237

**Review by jhult — DISMISSED** [review]
Also requesting to squash commits.

File: scripts/nuke-database.sh
Comment (n1tranquilla) [reply scripts/nuke-database.sh:3]: Yeah, I mean that would make sense. Although the intent is to use at your own risk.

File: scripts/download-blocks.sh
Comment (n1tranquilla) [reply scripts/download-blocks.sh:17]: That would be cool. Would these scripts be targeted to Granolas or to the broader community as well?

Comment (jhult) [reply scripts/download-blocks.sh:17]: I think targeting the broader community would be a good idea.

File: scripts/download-blocks.sh
Comment (trevorbernard) [inline scripts/download-blocks.sh:7]: This will fail. Blocks start at 2

File: scripts/download-blocks.sh
Comment (trevorbernard) [reply scripts/download-blocks.sh:17]: That's a different issue and shouldn't block this PR

File: scripts/nuke-database.sh
Comment (Isaac-DeFrain) [reply scripts/nuke-database.sh:3]: I will approve and we can add this later if needed.

**Review by Isaac-DeFrain — APPROVED** [review]
LGTM

**Review by trevorbernard — APPROVED** [review]

**Pull request merged**
